### DISCUSSION TITLE
番剧/电影页面功能失效

### DISCUSSION THREAD

**Discussion by AsCrovo** [the1812/Bilibili-Evolved#4574] [提问 / Q&A]
![image](https://github.com/the1812/Bilibili-Evolved/assets/118998425/95e6993d-16c6-4340-bc29-2dbbf914956b)
在番剧/电影播放时许多功能都会失效，如图，倍速拓展是打开的但并没有额外倍速选项，其他功能如双击全屏等也同样失效(对原先双击全屏的屏蔽倒是生效了)，但播放普通视频时能正常使用

**Comment by the1812** [comment]
#4571
